src/App.js

```diff
@@ -1,7 +1,46 @@
+import { NavLink, Route, Switch } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom/cjs/react-router-dom.min';
 import './App.css';
+import Main from './views/Main/Main';
+import Books from './views/Books/Books';
+import Characters from './views/Characters/Characters';
+import Films from './views/Films/Films';
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <BrowserRouter>
+        <header>
+          <NavLink to="/" data-testid="/" exact>
```

Comment:
The "data-testid" is not necessary on these NavLinks as they are not being tested. https://testing-library.com/docs/queries/bytestid/